2018/TPAC.md

```diff
@@ -44,9 +44,27 @@ Instead, Thursday and Friday will be full days of Community Group discussion. To
        1. [Tail calls](https://github.com/WebAssembly/tail-call) (Andreas Rossberg)
           - status update, resolution of open issues
           - poll: advance to stage 3?
+          - 30 min
        1. [C/C++ API](https://github.com/rossberg/wasm-c-api) (Andreas Rossberg)
           - new proposal presentation
           - poll: advance to stage 1?
+          - 30 min
+       1. [Threads](https://github.com/WebAssembly/threads) (Ben Smith, Andreas Rossberg, Conrad Watt)
+          - Update on memory model and spec
+          - [Race between memory.grow and access](https://github.com/WebAssembly/threads/issues/26#issuecomment-424742576)
+          - 1 hour
+       1. [Bulk Memory Operations](https://github.com/WebAssembly/bulk-memory-operations) (Ben Smith)
+          - poll: advance to stage 2?
+          - 30 min
+       1. [WebAssembly Content-Security-Policy](https://github.com/WebAssembly/content-security-policy) (Ben Titzer)
+          - [How does CSP affect postMessaging of modules](https://github.com/WebAssembly/content-security-policy/pull/16)
+          - 30-60 min
+       1. [WebAssembly Spec Update](https://github.com/WebAssembly/spec) (Ben Smith)
+          - 15 min
+       1. [JS BigInt to WebAssembly i64 Integration](https://github.com/WebAssembly/JS-BigInt-integration) (Dan Ehrenberg)
+          - 15 min
+       1. [ES6 Module Integration](https://github.com/WebAssembly/esm-integration) (Lin Clark)
+          - 30 min
```

Comment:
Any chance we can move JS BigInt to WebAssembly i64 Integration and ES6 Module Integration to Friday? cc @littledan @linclark?

Reply:
I'd be fine with it, though Friday has some topics that we may want to discuss for a while, so I thought it would be good to put smaller items on Thursday.

Reply:
I asked because I won't be available on Thursday and these are the subjects I'm working on, so it would be great if we could move them.

Reply:
OK, let's move them for now and we can wait to see if any others have conflicts.